parsely/parselyandroid/ParselyTracker.java

```diff
@@ -278,11 +336,13 @@ public void resetVideo() {
 
     /*! \brief Create an event Map
      *
-     *  @param url The canonical URL identifying the pageview/heartbeat
-     *  @param action Action kind to use (e.g. pageview, heartbeat)
+     *  @param url      The canonical URL identifying the pageview/heartbeat
+     *  @param action   Action kind to use (e.g. pageview, heartbeat)
```

Comment:
"Action kind" is inaccurate - I'd understand the "action kinds" to be `pageview` and `heartbeat`, where the events of kind "pageview" are `pageview` and `videostart` events. I think you mean "action name" or simply "action"